README.md

```diff
@@ -1,75 +1,100 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# COVID-19 STATS
 
+> This web application provides statistical insights on COVID-19 cases around the world. The application allows users to search for different countries and clicking a country would display all of the covid statistics for that country including its states, provinces and sub regions if available.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
+- React JS & Redux
+- CSS
+- JavaScript
+- Tailwindcss
 
-[Live Demo Link](https://livedemo.com)
+## Live Demo
+|Netlify cloud hosting|
+|----------------------|
+|[View live demo]()|
```

Comment:
- [ ] The live demo link that you provided is not working (Broken). Please deploy the application properly using Heroku, Netlify, or GitHub Pages.

![image](https://user-images.githubusercontent.com/68796232/194441006-aa4cbd37-5d76-4832-865a-9c3e05be572b.png)